core/src/main/java/me/mykindos/betterpvp/core/utilities/UtilEntity.java

```diff
@@ -180,4 +181,17 @@ public static void setViewRangeBlocks(@NotNull Display display, float blocks) {
     public static float getViewRangeBlocks(@NotNull Display display) {
         return display.getViewRange() * (float) (net.minecraft.world.entity.Entity.getViewScale() * 64.0);
     }
+
+    /**
+     * Sets fire to the damagee with damager as the source
+     * @param damagee The entity to set on Fire
+     * @param damager The entity that set the fire
+     * @param duration The duration to set the damagee on fire, in ms
+     */
+    public static void setFire(@NotNull Entity damagee, @NotNull Entity damager, long duration) {
+        EntityCombustByEntityEvent entityCombustByEntityEvent = new EntityCombustByEntityEvent(damagee, damager, (float) duration /1000L);
+        UtilServer.callEvent(entityCombustByEntityEvent);
```

Comment:
You can do entityCombustByEntityEvent = UtilServer.callEvent(new event...) to save yourself a line here

Reply:
done